content/en/docs/tasks/administer-cluster/subpath-env.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Using SubPath with Expanded Environment Variables 
+content_type: task
+min-kubernetes-server-version: v1.15
+---
+{{< feature-state for_k8s_version="v1.17" state="stable" >}}
+
+<!-- overview -->
+
+Use the `subPathExpr` field to construct `subPath` directory names from
+downward API environment variables. In other words, the `subPath` mounted by a pod may determined using variables, such as pod name. For example, you can create a folder for logs from each pod.
+
+The `subPath` and `subPathExpr` properties are mutually exclusive.
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+<!-- steps -->
+
+## Example manifest
+
+In this example, a pod uses `subPathExpr` to create a directory `pod1` within
+the `hostPath` volume `/var/log/pods`. All containers see this as simply `/logs`. Thus, logs may be collected from multiple pods in directories such as `/var/log/pods/pod1/` and `.../pods/pod2/`.
```

Comment:
Should we warn people that read-write `hostPath` mounts are usually not suited for production (and that [`local`](https://kubernetes.io/docs/concepts/storage/volumes/#local) might be a better option? The original page includes that warning, albeit quite a long way away from this text.

(Perhaps we could use a `local` volume in the example - but that sounds like a separate fix).

Reply:
I think i'll open an issue to make better examples using local.

Reply:
done! #27439